templates/crate_details.hbs

```diff
@@ -44,10 +44,14 @@
               <ul class="pure-menu-list">
                 {{#each releases}}
                 <li class="pure-menu-item">
-                  {{#if this.build_status}}
-                  <a href="/crate/{{../name}}/{{this.version}}" class="pure-menu-link">{{this.version}}</a>
+                  {{#if this.yanked}}
+                  <a href="/crate/{{../name}}/{{this.version}}" class="pure-menu-link warn" title="{{../name}}-{{this.version}} is yanked"><i class="fa fa-fw fa-warning"></i> {{this.version}}</a>
                   {{else}}
```

Comment:
Why is this an else? If it failed to build I feel like we should show both warnings.

Reply:
Double ⚠️ symbols, or just concatenate the hover messages?

Reply:
I think double :warning: is fine.